app/assets/javascripts/admin/controllers/admin-site-settings.js.es6

```diff
@@ -17,7 +17,9 @@ export default Ember.Controller.extend({
 
     if ((!filter || 0 === filter.length) && !this.get("onlyOverridden")) {
       this.set("visibleSiteSettings", this.get("allSiteSettings"));
-      this.transitionToRoute("adminSiteSettings");
+      if (this.get("categoryNameKey") === "all_results") {
```

Comment:
What is this fixing?

Reply:
I'm now reloading all the site settings asynchronously after each "save". This `transitionToRoute` was causing a change in route every time you saved a setting. Now it only transitions if necessary.